documentation/docs/libraries/python/how_to/_13_alias_output/_expected_output.md

```diff
@@ -1,26 +1,3 @@
 ```json
-{
-  "type": 4,
-  "amount": "50300",
-  "aliasId": "0xa5c28d5baa951de05e375fb19134ea51a918f03acc2d0cee011a42b298d3effa",
-  "stateIndex": 0,
-  "stateMetadata": "0x",
-  "foundryCounter": 0,
-  "unlockConditions": [
-    {
-      "type": 4,
-      "address": {
-        "type": 0,
-        "pubKeyHash": "0x7ffec9e1233204d9c6dce6812b1539ee96af691ca2e4d9065daa85907d33e5d3"
-      }
-    },
-    {
-      "type": 5,
-      "address": {
-        "type": 0,
-        "pubKeyHash": "0x7ffec9e1233204d9c6dce6812b1539ee96af691ca2e4d9065daa85907d33e5d3"
-      }
-    }
-  ]
-}
+{'type': 4, 'amount': '59200', 'aliasId': '0x0000000000000000000000000000000000000000000000000000000000000000', 'stateIndex': 0, 'stateMetadata': '0x68656c6c6f', 'foundryCounter': 0, 'unlockConditions': [{'type': 4, 'address': {'type': 0, 'pubKeyHash': '0x7ffec9e1233204d9c6dce6812b1539ee96af691ca2e4d9065daa85907d33e5d3'}}, {'type': 5, 'address': {'type': 0, 'pubKeyHash': '0x7ffec9e1233204d9c6dce6812b1539ee96af691ca2e4d9065daa85907d33e5d3'}}], 'features': [{'type': 0, 'address': {'type': 0, 'pubKeyHash': '0x7ffec9e1233204d9c6dce6812b1539ee96af691ca2e4d9065daa85907d33e5d3'}}, {'type': 2, 'data': '0x68656c6c6f'}], 'immutableFeatures': [{'type': 1, 'address': {'type': 0, 'pubKeyHash': '0x7ffec9e1233204d9c6dce6812b1539ee96af691ca2e4d9065daa85907d33e5d3'}}, {'type': 2, 'data': '0x68656c6c6f'}]}
```

Comment:
Can we not pretty print it like it was ?

Reply:
Yes, but then it's not like the output of the example :trollface:

Reply:
Then prettify the example :trollface:

Reply:
But yeah I get it, you can close if you want

Reply:
Done